include/flatbuffers/flatbuffers.h

```diff
@@ -1470,6 +1502,21 @@ class FlatBufferBuilder
     finished = true;
   }
 
+  // Allocates space for a vector of structures.
+  // Must be completed with EndVectorOfStructs().
+  template<typename T>
```

Comment:
See my comment elsewhere.. consistency > any style guide. If this is to be fixed, then all at once, and not in this PR.